src/integration-tests/introspector/introspectTest.sh

```diff
@@ -955,7 +1030,9 @@ checkWLVersionChecks
 # Check admin-server pod log and also call on-line WLST to check if
 # automatic and custom overrides are taking effect in the bean tree:
 
-checkOverrides
+if [ "${DOMAIN_SOURCE_TYPE}" != "FromModel" ] ; then
+  checkOverrides
```

Comment:
Several of the tested overrides are internal overrides - all of the 'ListenAddress' and 'Log' related ones.

Maybe we should test those on the model path?  

If so, the test needs to be made a little smarter so that internal overrides are tested whilst custom overrides are not.    Perhaps split them into two different "inputt" files - one for internal and one for external.  The combine the files into a temporary file for the PV path, and only copy one of the files over for the Model path.

Reply:
This will take a little bit more time, I need to take a look of what it is being tested first.

Reply:
More on this.  It looks like the test is not testing 'model updates'.   E.g. the custom overrides from the test  should be passed in as config map updates _and_ tested to make sure that they actually occurred.   So one alternative would be:

1) Modify the check bean program's 'inputt' file format so that fields that are from internal overrides are tagged to indicate they always come from an internal override.
2) Modify the mbean checker so that it acts differently in FromModel mode and reacts to the tag differently accordingly.   E.g. the tag makes no difference in PV mode, but in FromModel mode, it expects things that don't have the 'IO' internal-override tag to have the expected value in both the live and the static mbean trees, instead of just in the live tree.

Reply:
The current test is only one pass, i.e. no update test case.  This is one area that I agreed we should spend some effort into.

Reply:
I'm not advocating a second pass.  Just a single pass that includes verification that the settings from the config map took effect, and that verifies the internal overrides took effect.

Reply:
Yes, it's already there.  The configmap is actually coded in a way that works for both non mii case

Reply:
Where's the verification that the listen address, etc, are as expected?

Reply:
Not for listening address, but the data sources are there.  We can change the override to check the internal ones only for mii

Reply:
Please do.  See my original suggestion at the beginning of this thread.

Reply:
Looks good now.